### PR TITLE
refactor: Replaced deprecated Gtk::StockID in ResolutionDial

### PR DESCRIPTION
**Pull request by ice0**
Before:
Screenshot_79

After:
Screenshot_80

### CODE REVIEW

File: synfig-studio/src/gui/canvasview.cpp [hunk 1]
```diff
@@ -520,7 +521,7 @@ CanvasView::CanvasView(etl::loose_handle<Instance> instance,etl::handle<CanvasIn
 	statusbar                (manage(new class Gtk::Statusbar())),
 	progressbar              (manage(new class Gtk::ProgressBar())),
 	toggleducksdial          (Gtk::IconSize::from_name("synfig-small_icon_16x16")),
-	resolutiondial           (Gtk::IconSize::from_name("synfig-small_icon_16x16")),
+	resolutiondial           (new studio::ResolutionDial()),
```
Comment (rodolforg) [inline synfig-studio/src/gui/canvasview.cpp:524]: Where is the paired `delete` for this `new`?

**Review by rodolforg — CHANGES_REQUESTED** [review]

File: synfig-studio/src/gui/dials/resolutiondial.cpp [hunk 2]
```diff
@@ -54,11 +51,31 @@ using namespace studio;
...
-	init_button(increase_resolution, size, Gtk::StockID("synfig-increase_resolution"), _("Increase Resolution"), _("Increase Display Resolution"));
-	init_button(decrease_resolution, size, Gtk::StockID("synfig-decrease_resolution"), _("Decrease Resolution"), _("Decrease Display Resolution"));
+	// For label left/right padding
+	button.set_name("low-resolution");
+
+	button.set_label(label);
+	button.set_tooltip_text(tooltip);
+	button.set_is_important(true);
+	button.show();
+}
```
Comment (rodolforg) [inline synfig-studio/src/gui/dials/resolutiondial.cpp:73]: This should be in section `P R O C E D U R E S`

**Comment by rodolforg** [comment]
"Resolution" is misspelled in commit and PR message

File: synfig-studio/src/gui/canvasview.cpp (hunk 1 above)
Comment (ice0) [reply synfig-studio/src/gui/canvasview.cpp]: Done

File: synfig-studio/src/gui/dials/resolutiondial.cpp (hunk 2 above)
Comment (ice0) [reply synfig-studio/src/gui/dials/resolutiondial.cpp]: Done

**Review by rodolforg — APPROVED** [review]

**Pull request merged**